pkg/controller/handler.go

```diff
@@ -1,75 +1,46 @@
 package controller
 
 import (
-	"github.com/containous/maesh/pkg/k8s"
 	"github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
 )
 
+// configMessageChanRebuild rebuild.
+var configMessageChanRebuild = struct{}{}
```

Comment:
WDYT of just sending a `struct{}{}` but not having a variable for it? If the channel names are explicit enough, we should not need to have a variable for this.